src/mapping.ts

```diff
@@ -1,8 +1,14 @@
-import { dataSource } from '@graphprotocol/graph-ts';
-import { DelegateChanged, DelegateVotesChanged } from '../generated/Token/Token';
+import { Address, dataSource } from '@graphprotocol/graph-ts';
+import { DelegateChanged, DelegateVotesChanged } from '../generated/templates/GenericERC20Votes/Token';
 import { toDecimal, getDelegate, getGovernance, BIGINT_ZERO } from './helpers';
+import { ContractDeployed } from "../generated/GeneralPurposeFactory/GeneralPurposeFactory";
```

Comment:
Can you use single quote everywhere?
```suggestion
import { ContractDeployed } from '../generated/GeneralPurposeFactory/GeneralPurposeFactory';
```